OutOfSchool/OutOfSchool.WebApi/Startup.cs

```diff
@@ -191,7 +192,12 @@ public static async Task AddApplicationServices(this WebApplicationBuilder build
                         Duration = cacheProfilesConfig.PublicDurationInSeconds,
                     });
             })
-            .AddNewtonsoftJson()
+
+            // to handle self-references during serialization
+            .AddNewtonsoftJson(options =>
```

Comment:
are you sure it won't be an issue ?
I mean, we should find the root cause but not ignore the issue